README.md

```diff
@@ -16,25 +53,48 @@ gem 'health_cards'
 
 And then execute:
 
-    $ bundle install
+```
+ $ bundle install
+```
 
 Or install it yourself as:
 
-    $ gem install health_cards
+```
+ $ gem install health_cards
+```
+
+### Usage
+
+```ruby
+# Create issuer
+private_key = HealthCards::Key.generate_key
+issuer = HealthCards::Issuer.new(key: private_key)
```

Comment:
Get rid of this code snippet and just point to `USAGE.md`. Having this documentation in two different places is just an opportunity to have something go out of date.

Reply:
fixed